### ISSUE TITLE
winrm fails to provision inline shell to windows 10 guest

### ISSUE THREAD

**Issue by dafyddj** [mitchellh/vagrant#6055]
vagrant 1.7.4 
Host: OS X 10.10.4
Guest: Windows 10 build 10162

This works using a Windows 8.1 guest.

Line of interest
```
/opt/vagrant/embedded/gems/gems/winrm-fs-0.2.0/lib/winrm-fs/core/command_executor.rb:59:in `assert_command_success': uninitialized constant WinRM::FS::Core::CommandExecutor::WinRMUploadError (NameError)
```

Debug output: https://gist.github.com/dafyddj/f347cb99034786f2d282

**Comment by taliesins** [comment]
This is resolved in my comment with a patch in #6060

**Comment by mitchellh** [comment]
Fixed by #6060
